src/components/wrapper/index.js

```diff
@@ -12,7 +13,7 @@ import './wrapper.css';
 /**
  * Main app container. Handles showing/hiding the sidebar nav, and theme classes.
  */
-export const Wrapper = ({ loading, theme }) => (
+export const Wrapper = ({ loading, theme, showLargeGraphWarning }) => (
```

Comment:
Looks like `showLargeGraphWarning` isn't being used here, so think better to remove? Either that or go with something like `visible={showLargeGraphWarning}`.

Reply:
This has been taken out now hence resolving this